app/views/organization_timeline_entries/index.html.erb

```diff
@@ -19,12 +22,25 @@
     <% @entries.each do |entry| %>
       <% if can?(:read, entry) %>
         <tr>
-          <td class="hidden-xs"><%= link_to date_and_time(entry.started_at), edit_organization_timeline_entry_path(entry) %></td>
+          <td class="hidden-xs"><%= link_to date_and_time(entry.started_at), organization_timeline_entry_path(entry) %></td>
           <td class="hidden-xs"><%= date_and_time entry.ended_at unless entry.ended_at.blank? %></td>
           <td class="hidden-xs"><%= format_downtime entry.duration %></td>
+          <% if can?(:update, OrganizationTimelineEntry) or can?(:destroy, OrganizationTimelineEntry) %>
+            <td class="hidden-xs">
+              <% if can?(:update, OrganizationTimelineEntry) %>
+                <%= link_to t('.edit', :default => t("helpers.links.edit")),
+                        edit_organization_timeline_entry_path(entry), :class => 'btn btn-xs btn-primary' %>
+              <% end %>
+              <% if can?(:destroy, OrganizationTimelineEntry) %>
+                <%= link_to t('.destroy', :default => t("helpers.links.destroy")),
+                        organization_timeline_entry_path(entry),
+                        :method => :delete,
```

Comment:
The links themselves work.  When you go to the show view and then click back things break.  The reason is that you are using "organization_timeline_entry_path" instead of "downtime_path".  It works in this case because the forms use the "downtime_path" link.  But we should never ever use any form of "organization_timeline_entry_path".